notebooks/demo/centrality_patentsview.ipynb

```diff
@@ -0,0 +1,590 @@
+    "    df4_styler = pr_top.style.set_table_attributes(\"style='display:inline'\").set_caption('PageRank').hide(axis='index')\n",
+    "    df5_styler = ev_top.style.set_table_attributes(\"style='display:inline'\").set_caption('EigenVector').hide(axis='index')\n",
+    "\n",
+    "    display_html(df1_styler._repr_html_()+df2_styler._repr_html_()+df3_styler._repr_html_()+df4_styler._repr_html_()+df5_styler._repr_html_(), raw=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After running the centrality algorithms, this function displays the supplied graph. The dataframe from any of the centrality calculations is also supplied along with the number of the highest ranking nodes we want to emphasize with labels and scale with size based on the centrality value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def draw_centrality_graph(graph_df, number_to_label, cent_df):\n",
+    "    import networkx as nx\n",
+    "    import matplotlib.pyplot as plt\n",
+    "\n",
+    "    top_ones = cent_df.sort_values(by=cent_df.columns[0], ascending=False).head(number_to_label)[cent_df.columns[1]].to_pandas()\n",
+    "    print(top_ones)\n",
+    "    sizes = []\n",
+    "    nodes = []\n",
+    "\n",
+    "    plt.rcParams['figure.figsize'] = [12, 8]\n",
+    "    nx_g = nx.from_pandas_edgelist(graph_df.to_pandas())\n",
+    "\n",
+    "    for node in nx_g.nodes():\n",
+    "        cent_value =cent_df.loc[cent_df['vertex'] == node, cent_df.columns[0]].values[0]\n",
+    "        nodes.append(node)\n",
+    "        sizes.append((cent_value[0]*3000+10).item())\n",
+    "\n",
+    "    pos = nx.spring_layout(nx_g)\n",
+    "    labels = {node: node for node in top_ones}\n",
+    "    nx.draw(nx_g, pos=pos, nodelist=nodes,font_color='red' ,node_size=sizes,node_color='grey', node_shape='o', linewidths=2, labels=labels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This function executes the five centrality algorithms."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Compute Centrality\n",
+    "# the centrality calls are very straightforward with the graph being the primary argument\n",
+    "# we are using the default argument values for all centrality functions except where noted.\n",
+    "\n",
+    "def compute_centrality(_graph, highest_degree=2000) :\n",
+    "\n",
+    "    # Compute Degree Centrality\n",
+    "    _d = cugraph.degree_centrality(_graph)\n",
+    "    print(\"Degree Centrality done\")\n",
+    "\n",
+    "    # Compute the Betweenness Centrality\n",
+    "    # The k value is needed in large graphs as by default it will calculate the for every pair in the graph.\n",
```

Comment:
I think there is a missing word in this sentence between `the` and `for`. And I you can rephrase it slightly differently for better clarity. As you already know, the between centrality score of a vertex is a measure of the number of shortest paths that passes through it. Hence for large graphs, this can be really expensive because the algorithm will need to compute the all-pair shortest path. Setting a value of `k` limits the number of vertices involved in the all pairs shortest computation for `bc`. As a suggestion you can rephrase it along theses lines: 

> The k value is needed in large graphs as by default, it will calculate the betweenness centrality score based on every possible pairs in the graph. Limiting the number of vertices involved in the shortest path search leads to better runtime.

The second sentence is optional if you want to more descriptive